cmd/kubectl-gadget/version.go

```diff
@@ -35,7 +40,47 @@ func init() {
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Show version",
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(version)
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Println("Client version:", version)
+
+		client, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
+		if err != nil {
+			return commonutils.WrapInErrSetupK8sClient(err)
+		}
+
+		opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
+		pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
+		if err != nil {
+			return commonutils.WrapInErrListNodes(err)
```

Comment:
I just realized we were incorrectly using this error when there is an error listing the **pods** but it actually talks about **nodes** (`failed to list nodes`) 🤦🏽. I think we can just make it more generic, something like `failed to list Kubernetes resources`. WDYT?

Reply:
Yep, that is a ListNodes error message, not a ListPods.
About generalizing the errors: Depends on how we would like to proceed with these kind of things. More generic or having many specific messages

Reply:
We can manage this in another PR 🙂